userops_ext_test.go

```diff
@@ -198,6 +231,15 @@ func mockUserOperationWithIntentInCallData() *UserOperation {
 	return userOp
 }
 
+func mockUserOperationWithCrossChainIntentInCallData() *UserOperation {
+	userOp := new(UserOperation)
+	intentJSON := mockCrossChainIntentJSON()
+
+	userOp.CallData = []byte(intentJSON)
```

Comment:
- **CallData Format:**
    - [2 bytes opType (0xFFFF)]
    - [2 bytes length of intent JSON]
    - `[Intent JSON]`
    - [1 byte hash list length (N)]
    - [**Hash List Entry**]